spec/index.md

```diff
@@ -1485,7 +1501,7 @@ The comment after each function lists from where these functions may be invoked:
 
 -   `F`: from `canister_inspect_message`
 
--   `T`: from *system task* (`canister_heartbeat` or `canister_global_timer`)
+-   `T`: from *system task* (`canister_heartbeat` or `canister_global_timer` or `canister_on_low_wasm_memory`)
```

Comment:
If we make this of type `T`, then a canister could update the threshold in a setting and allocate a new page, which should make the hook trigger again. So if we guarantee *immediate* execution, one can build a chain of calls to the hook. So we need to make sure that *immediate* may mean that it executes as part of the next execution round (just that nothing else runs in between).